CriteoPublisherSdk/Sources/Configuration/CR_IntegrationRegistry.m

```diff
@@ -80,4 +81,21 @@ - (BOOL)isAdMobMediationPresent {
          NSProtocolFromString(@"GADCustomEventBanner") != nil;
 }
 
+- (CR_CacheAdUnitArray *)filterAdUnits:(CR_CacheAdUnitArray *)units {
```

Comment:
The naming is kind of weird in the context of the CR_IntegrationRegistry.
From this class, I would either expect a method indicating which integrations are supported for the given ad unit (or a predicate taking an ad unit corresponding of your block), or a method that keep only supported ad units.

Yours is doing that but the naming is not perfect IMHO. What do you think about `filterSupportedAdUnits` or `keepSupportedAdUnits`, or `isAdUnitSupported` (for predicate)